django_cron/backends/lock/database.py

```diff
@@ -5,13 +5,13 @@
 
 class DatabaseLock(DjangoCronJobLock):
     """
-    Locking cron jobs with database. Its good when you have not parallel run and want to make sure 2 jobs won't be
+    Locking cron jobs with database. It's good when you have not parallel run and want to make sure 2 jobs won't be
```

Comment:
"have not parallel run" doesn't read right.

```suggestion
    Locking cron jobs with database. It's good when you have not run cron jobs in parallel and want to make sure 2 jobs won't be
```